js/gated-content/src/components/personal-training/meeting/MeetingPlayer.vue

```diff
@@ -20,6 +21,13 @@
       </video>
     </div>
     <div class="video-wrapper local" :class="{connected: localMediaStream !== null}">
+      <AudioIndicator :media-stream="localMediaStream" :mic-enabled="isMicEnabled" />
+      <div class="video-state-indicator">
```

Comment:
Since, we've created Audio indicator above, we can easily develop small component for video indicator and put it everywhere on the app.